.zuul.yaml

```diff
@@ -0,0 +1,11 @@
+- project:
+    vars:
+      functest_project_name: "eu-de_zuul_go"
+    check:
+      jobs:
+        - golangci-lint
+        - golang-make-functional
```

Comment:
Using credcential information in `check` can be insecure. Please move functional to gate.